lib/elixir/lib/kernel.ex

```diff
@@ -932,6 +937,11 @@ defmodule Kernel do
   Removes the first occurrence of an item on the left list
   for each item on the right.
 
+  The complexity of `a -- b` is proportional to `length(a) * length(b)`,
+  meaning that it will be very slow if both `a` and `b` are long lists.
```

Comment:
I understand why you don't like the term "very slow", but I worry that someone will read "grow in size" and not worry about it because their lists don't "grow" I get what you mean there, in that that's the asymptotic behaviour of the algorithm, but the performance of any given use of `--` just has to do with the size of each list at the time it's used.

Maybe something along the lines of "meaning that care should be taken when both `a` and `b` are long lists."